torchvision/csrc/ROIAlign.h

```diff
@@ -147,7 +105,46 @@ class ROIAlignFunction : public torch::autograd::Function<ROIAlignFunction> {
   }
 };
 
-at::Tensor roi_align(
+// TODO: There should be an easier way to do this
```

Comment:
Can't we register a fallback kernel that raises an error on double-backward? So that we don't have to implement a dummy double-backwards kernel for all the ops.

Reply:
This would indeed be the right thing to do in core library. There will be some BC consequences though